website/docs/getting_started.html.markdown

```diff
@@ -0,0 +1,224 @@
+---
+layout: "google"
+page_title: "Getting Started with the Google provider"
+sidebar_current: "docs-google-provider-getting-started"
+description: |-
+  Getting started with the Google Cloud Platform provider
+---
+
+# Getting Started with the Google Provider
+
+## Before you begin
+
+* You'll want to have created an account and project in the [Google Cloud Console](https://console.cloud.google.com/)
```

Comment:
This is a teeny bit wordy, and I think it's not super important to say to create an account. maybe just something like "Create a project in [...]"